common/changes/@microsoft/rush/bartvandenende-wm-support-json-schema-draft-07_2024-05-15-16-44.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Update test assertions for schema validation failures.",
+      "type": "none"
```

```suggestion
      "comment": "",
      "type": "none"
```

This isn't relevant to a consumer of the package.